apps/speed.c

```diff
@@ -1697,7 +1715,7 @@ int speed_main(int argc, char **argv)
             evp_md = NULL;
             evp_cipher = EVP_get_cipherbyname(opt_arg());
```

Comment:
It seems like we should make a similar change here too.

Reply:
If you mean evp_cipher, I agree with you. But I want to fix some more performance issues for digests.

Reply:
Done. Thanks!